Godeps/Godeps.json

```diff
@@ -0,0 +1,9 @@
+{
+	"ImportPath": "github.com/lilic/mountinfo",
```

Comment:
same for vendor/vendor.json

Also I guess we don't need both Godeps.json and vendor.json but only one of them